lib/confluence/api_velocity.es6.js

```diff
@@ -113,11 +113,14 @@ foam.CLASS({
                     });
                     continue;
                   }
-                  browser.interfaces.select().then((arraySink) => {
+                  let tempPromise = prevIfaceSelect;
```

Comment:
Please document control flow for lines 116 - 118. Document what we're trying to do in parallel and in series.